Firmware/ultralcd.cpp

```diff
@@ -1833,19 +1833,19 @@ switch(eFilamentAction)
 
 void mFilamentBack()
 {
-    if (eFilamentAction == FilamentAction::AutoLoad ||
-        eFilamentAction == FilamentAction::Preheat ||
-        eFilamentAction == FilamentAction::Lay1Cal)
-    {
-        // filament action has been cancelled
-        eFilamentAction = FilamentAction::None;
-    }
+    // filament action has been cancelled
+    eFilamentAction = FilamentAction::None;
```

Comment:
The original idea was that if the current action is "Load", and you exit the menu, it stays "Load" until it is finally cleared by unload_filament().

This is why this checks for AutoLoad/Preheat/Lay1Cal: in the comments these are the states which can be "aborted" in the preheat menu.

Reply:
The menu should return to full automatically as soon as the load is complete.

Reply:
@gudnimg would it make sense to keep `mFilamentBack()` as it has been before and just set `eFilamentAction = FilamentAction::None;` in `mFilamentDone()`?

I haven't tried myself, I'm just trying to understand what's going on.

Reply:
I still think this was incorrect. The filament action should only be cleared by the actual move, not by the menu, otherwise we re-introduce the issue fixed in #3564 where exiting the menu will allow actions such as autoload while autoload is still taking place.

Reply:
Ah that make sense to me. I agree with @wavexx I can try find an alternative fix.